### PR TITLE
Hig 4l 2017

### PR DESCRIPTION
**Pull request by skeshri**
Hi,

I have added cards for VBF production of 125 GeV Higgs signal in 4l channel along with Background only and Background+Signal+interference. This is just for 2e2mu channel. More will come once they are verified by experts.

Thanks

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/ZZTo2L2Q01j_5f_NLO_FXFX/ZZTo2L2Q01j_5f_NLO_FXFX_madspin_card.dat [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+#
+#directory for gridpack mode
+set ms_dir ./madspingrid
+
+#initialization parameters
+ set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
+# set BW_cut 15                # cut on how far the particle can be off-shell
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/ZZTo2L2Q01j_5f_NLO_FXFX/ZZTo2L2Q01j_5f_NLO_FXFX_madspin_card.dat]: please cleanup the datacards following these instructions https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T

File: bin/Phantom/cards/production/13TeV/HIG_VBF_4l_2017/Higgs_125_2e2mu_BSI.cfg [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+[parameters]
+# parameters for the phantom config file
+# parameters in this cfg file will be replaced in the phantom cfg template.
+
+# NECESSARY PARAMETER
+# the full list of available pdf is found in the folder stored 
+# in the $LHAPDF_DATA_PATH environment variable of the CMSSW release setup
+# working with LHAPDF6 only
+PDFname        = NNPDF31_lo_as_0130
```
Comment (perrozzi) [inline bin/Phantom/cards/production/13TeV/HIG_VBF_4l_2017/Higgs_125_2e2mu_BSI.cfg]: we recommend to use NNLO PDF for everything @govoni

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/ZZTo2L2Q01j_5f_NLO_FXFX/ZZTo2L2Q01j_5f_NLO_FXFX_proc_card.dat [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import model loop_sm-ckm_no_b_mass
+#switch to diagonal ckm matrix if needed for speed
+#import model loop_sm-no_b_mass
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/ZZTo2L2Q01j_5f_NLO_FXFX/ZZTo2L2Q01j_5f_NLO_FXFX_proc_card.dat]: remove unused import

Comment (skeshri) [reply bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/ZZTo2L2Q01j_5f_NLO_FXFX/ZZTo2L2Q01j_5f_NLO_FXFX_proc_card.dat]: Dear @perrozzi I have removed unused import line. 
I don't know about the conflict under.

**Comment by govoni** [comment]
@skeshri could you please run the gridpack integrity check on the gridpacks you produced, following the instrutions here?

https://twiki.cern.ch/twiki/bin/view/CMS/PhantomGeneratorCMS#the_gridpack_integrity_verificat

**Comment by skeshri** [comment]
@govoni I have run integrity test and the results are in the public directory.
/afs/cern.ch/work/s/skeshri/public/forPietro/IntegrityTest
Total Cross section value is reasonable.

**Comment by govoni** [comment]
@perrozzi @skeshri  are the CMSSW release and architecture compatibile with the Fall17 production?

**Comment by skeshri** [comment]
@govoni I think architecture is fine, but the release 931 should be used. I have cross checked again. Also there is some doubt related to pdf. I think recommendation for FallMC17 to use nnlo pdf. Here I have used LO. 
I have used LO because, it was used in earlier production. What should be use now?

**Comment by govoni** [comment]
@skeshri concerning the architecture: it goes with the release, therefore if you use CMSSW_9_3_0 you need to use the slc6_amd64_gcc630 architecture (see the comments in the example cfg file, line 24).
Concerning PDF: the default choice is NNLO in fact, as in the example cfg file. If in your analysis group you decide to go for LO PDFs for some reason of yours, you may argue of course, so I suggest you to check with them.

**Comment by skeshri** [comment]
@govoni I discussed about the PDF with Heshy. He suggested as Phantom is LO so to match with the Generator we should use LO pdf.

**Comment by perrozzi** [comment]
this PR has conflicts, can you please solve?

**Comment by skeshri** [comment]
Dear All, 
I'm closing this PR and will make a fresh one. I'm getting trouble in resolving conflicts. 
Will create a fresh PR soon.
Thanks

**Pull request closed without merging**